examples/render-helm-chart-crds/.expected/config.yaml

```diff
@@ -0,0 +1,17 @@
+# This test has customresourcedefinition_workspaces.app.terraform.io.yaml in its
```

Comment:
It seems this file only contains comments.
Should we delete this file given that we have an issue opened for it?

Reply:
deleted